src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -1,21 +1,77 @@
 ﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="ilmerge.props" />
 
   <PropertyGroup>
     <Description>NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.</Description>
     <TargetFrameworks>net45</TargetFrameworks>
-    <TargetFramework />
-    <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1572</NoWarn>
+    <TargetFramework></TargetFramework>
+    <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1572;NU5128</NoWarn>
```

Comment:
did you rebase with today's changes?
I merged a pack PR that'll fail the build if a warning was elevated to an error...a bug fixedin 5.7, but not in 5.6. 

If there's anything missing it might break the official CI build.

Reply:
will do.

Reply:
rebased now...we'll verify that CI likes it. thx.